cmd/skopeo/utils.go

```diff
@@ -104,6 +113,8 @@ func imageFlags(global *globalOptions, shared *sharedImageOptions, flagPrefix, c
 	fs := pflag.FlagSet{}
 	fs.StringVar(&opts.sharedBlobDir, flagPrefix+"shared-blob-dir", "", "`DIRECTORY` to use to share blobs across OCI repositories")
 	fs.StringVar(&opts.dockerDaemonHost, flagPrefix+"daemon-host", "", "use docker daemon host at `HOST` (docker-daemon: only)")
+	fs.IntVar(&opts.maxRetry, "retry-times", 0, "the number of times to retry")
+	fs.DurationVar(&opts.retryDelay, "retry-delay", 5*time.Second, "time to wait before retry execution")
```

Comment:
We might want to use exponential backoff — so, I’m not sure how much to commit to these exact semantics of `retry-delay`.

Reply:
Dose this need to be a `max delay interval`, so the delay time will be the min(`expotional wait time`, `max delay interval`)

Reply:
I was thinking maybe “initial retry delay”, with the further retries using a longer (documented or undocumented?) delay; that way the user can choose the delay for the hopefully common case of only one retry, and the total time (via the global timeout), without being promised a specific number of retries.

But I’m not at all sure, others may come up with much better ideas.